src/background/actions/urls.ts

```diff
@@ -179,6 +180,9 @@ export async function resolveUrl(url: string): Promise<ResolvedUrl> {
         return createUnexpectedError(e, "error while trying to fetch metadata file");
       }
 
+      if (metadataFileType.mediaType === METADATA_FILE_TYPES[0].mediaType)
```

Comment:
Don't rely on the ordering here, this is fragile. If you'd like to check just for `.torrent` files, I suggest you go to the file where `METADATA_FILE_TYPES` is defined and do something like

```ts
export const TORRENT_METADATA_FILE_TYPE = { ... };
export const ALL_METADATA_FILE_TYPES = [ TORRENT_METADATA_FILE_TYPE, ... ];
```

In this way we can be sure that everyone agrees on what constitutes a torrent file.

Reply:
OK